Abort does not compile:
| error: the crate `panic_abort` does not have the panic strategy `abort`

unwind seems to be recommended:
From https://github.com/meta-rust/meta-rust/issues/343
  Using panic abort leaves no cookie crumbs in the syslog, which is why unwind is preferred.
diff -aurN a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml	2024-05-02 09:13:24.846786317 +0200
+++ b/Cargo.toml	2024-05-02 10:40:19.612420380 +0200
@@ -16,10 +16,10 @@
 resolver = "2"
 
 [profile.dev]
-panic = 'abort'
+panic = 'unwind'
 
 [profile.release]
-panic = 'abort'
+panic = 'unwind'
 # Release builds will have full symbols. The packaging phase will strip symbols from binaries and
 # make them available in a separate package.
 # Notes: Snaps don't have a good story for debug symbols, so for now we'll override this setting in
